File: solutions/36-riyad/AgeofVillagers/AgeofVillagers/AgeofVillagers/InuitHunters.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Text;

namespace AgeofVillagers
{
    public class InuitHunters : INation
    {

        Pen pen = new Pen(Color.Black);
        public string DrawHouse(Graphics g, Point p)
        {
            IunitHuntersHouse iHouse = new IunitHuntersHouse(p);
            iHouse.Paint(g, pen);
            return "Igloo";
        }

        public void DrawTree(Graphics g, Point p)
        {
            NulTree nulTree = new NulTree();
        }

        public void DrawWaterSource(Graphics g, Point mid)
        {
            NulTree nulTree = new NulTree();
        }

        public Color TerrainColor()
        {
            return System.Drawing.Color.White;
        }
    }
}

File: solutions/85-joy/src/project/Utilities/InuitHunters.java
package project.Utilities;

import javafx.scene.shape.Shape;

import java.util.ArrayList;

public class InuitHunters implements INation{
    private  String TerrainColor = "WHITE";

    @Override
    public ArrayList<Shape> DrawHouse(double x1, double y1) {
        return null;
    }

    @Override
    public ArrayList<Shape> DrawTree(double x1, double y1) {
        return null;
    }

    @Override
    public ArrayList<Shape> DrawWaterSource(double x1, double y1) {
        ArrayList<Shape> WaterSource = new ArrayList<Shape>();
        
    }

    @Override
    public String getTerrainColor() {
        return TerrainColor;
    }
}
